respy/tests/test_integration.py

```diff
@@ -443,29 +335,25 @@ def test_12(self):
         types and initial conditions.
         """
         # Sample one task
-        resources = []
-        resources += ["kw_data_one.ini", "kw_data_two.ini", "kw_data_three.ini"]
-        resources += ["kw_data_one_types.ini", "kw_data_one_initial.ini"]
+        results = {
+            "kw_data_one": 10.45950941513551,
+            "kw_data_two": 45.04552402391903,
+            "kw_data_three": 74.28253652773714,
+            "kw_data_one_types": 9.098738585839529,
+            "kw_data_one_initial": 7.965979149372883,
+        }
 
-        fname = np.random.choice(resources)
+        resources = list(results.keys())
 
-        rslt = None
-        if "one.ini" in fname:
-            rslt = 10.45950941513551
-        elif "two.ini" in fname:
-            rslt = 45.04552402391903
-        elif "three.ini" in fname:
-            rslt = 74.28253652773714
-        elif "one_types.ini" in fname:
-            rslt = 9.098738585839529
-        elif "one_initial.ini" in fname:
-            rslt = 7.965979149372883
+        fname = np.random.choice(resources)
 
         # This ensures that the experience effect is taken care of properly.
         open(".restud.respy.scratch", "w").close()
 
+        base_path = join(TEST_RESOURCES_DIR, fname)
```

Comment:
pathlib paths are joined with ``TEST_RESOURCES_DIR / fname``.

Reply:
This is too large for now. See #156